bootrom/bootrom.asm

```diff
@@ -1,34 +1,43 @@
 SECTION "bootrom", ROM0[$0000]
-main:
-.loop:
     ; Init stackpointer
-    ld SP, $FFFE
+    ld sp, $fffe
 
-    ; Enable LCD and background tilemap
-    ld A, $91
-    ld [$FF00+$40], A
-
-    ; Set color palette to 11111100
-    ld A, $fc
-    ld [$FF00+$47], A
+    ; Erase VRAM - it is filled with garbage on startup on hardware
+    xor A
+    ld HL,$9fff
+Erase:
+    ld [HL-],A
+    bit 7,H
+    jr nz, Erase

```

Comment:
I can recognize, that this is from the official DMG bootrom. We cannot include anything in the project, that is obviously copied like that. Both so we don't have any case for DMCA notices, but also morally, that we don't copy-paste code from any source. No matter how trivial it is.